release-notes.txt

```diff
@@ -0,0 +1,173 @@
+BISDN Linux
+4.7.0 Release Notes
+Date: 2022-08-03
+
+! New in this release
+- Yocto: Updated Yocto from 3.1 (dunfell) to 4.0 (kirkstone).
+  This includes major version updates for most OS components. For a detailed
+  list of changes, see [1] and the release notes for 3.2 to 3.4.
+
+- Linux kernel: Updated to 5.15.56.
+- FRR: Updated to 8.3.
```

Comment:
This needs an update.

Reply:
Dropped it, and added the revert commit to meta-switch.